examples/default.yaml

```diff
@@ -82,6 +82,27 @@ ssh:
   # 🟢 Builtin default: false
   forwardAgent: null
 
+ca_certs:
```

Comment:
The naming convention would be `caCerts`.

We only have a single exception for `cpuType.x86_64` because that is the string used as the `arch` value in other places.

Reply:
And should be moved to the `FURTHER ADVANCED CONFIGURATION` section

Reply:
Fixed.